src/test/java/com/microsoft/sqlserver/jdbc/preparedStatement/BatchExecutionWithBulkCopyTest.java

```diff
@@ -538,7 +539,7 @@ public void testIllegalNumberOfArgNoColumnList() throws Exception {
             pstmt.executeBatch();
             throw new Exception("Test did not throw an exception when it was expected.");
```

Comment:
Also fetch these Strings in all places from TestResource `R_expectedExceptionNotThrown`

Reply:
cool, done.